crmsh/ui_cluster.py

```diff
@@ -301,9 +301,8 @@ def looks_like_hostnames(lst):
                             help='Answer "yes" to all prompts (use with caution, this is destructive, especially those storage related configurations and stages. The /root/.ssh/id_rsa key will be overwritten unless the option "--no-overwrite-sshkey" is used)')
         parser.add_argument("-n", "--name", metavar="NAME", dest="cluster_name", default="hacluster",
                             help='Set the name of the configured cluster.')
-        parser.add_argument("-N", "--nodes", metavar="NODES", dest="nodes",
+        parser.add_argument("-N", "--nodes", metavar="NODES", dest="nodes", action="append", default=[],
                             help='Additional nodes to add to the created cluster. May include the current node, which will always be the initial cluster node.')
```

Comment:
Cosmetic improvement for the grammar in UI

```suggestion
                            help='The member node of the cluster. Note: the current node is always get initialized during bootstrap in the beginning.')
```